knotx-gateway/src/main/java/io/knotx/gateway/impl/RequestProcessorKnotProxyImpl.java

```diff
@@ -0,0 +1,71 @@
+/*
+ * Copyright (C) 2016 Cognifide Limited
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.knotx.gateway.impl;
+
+import java.util.NoSuchElementException;
+import java.util.Set;
+
+import io.knotx.dataobjects.ClientResponse;
+import io.knotx.dataobjects.KnotContext;
+import io.knotx.knot.AbstractKnotProxy;
+import io.netty.handler.codec.http.HttpResponseStatus;
+import io.vertx.core.buffer.Buffer;
+import io.vertx.core.http.HttpHeaders;
+import rx.Single;
+
+public class RequestProcessorKnotProxyImpl extends AbstractKnotProxy {
+
+  private final static String RESPONSE = "{\"message\":\"This is a sample custom flow response\"}";
```

Comment:
Should this class be a part of gateway implementation? It rather looks like an example. Maybe in such case this should go to `knotx-examples` or even to wiki docs?